example_sidecars/label-sidecar/cmd/label-sidecar.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	flag "github.com/spf13/pflag"
+	corev1 "k8s.io/api/core/v1"
+	v12 "k8s.io/apiserver/pkg/quota/v1"
+	"k8s.io/kubernetes/pkg/quota/v1/evaluator/core"
+	"k8s.io/utils/clock"
+	"kubevirt.io/application-aware-quota-api/libsidecar"
+)
+
+const (
+	// Double Calculate double amount of usage for pod
+	Double configName = "double"
+	// Triple Calculate triple amount of usage for pod
+	Triple        configName = "triple"
+	labelAppLabel            = "label-app"
+)
+
+type configName string
+
+type labelCalculator struct {
+	evaluatorConfig configName
+}
```

Comment:
You can add the following to make sure the struct is compliant with the interface:
```go
var _ = libsidecar.SidecarCalculator(&labelCalculator{})
```

Reply:
Done :)